fastlane/lib/fastlane/actions/reset_simulator_contents.rb

```diff
@@ -2,6 +2,10 @@ module Fastlane
   module Actions
     class ResetSimulatorContentsAction < Action
       def self.run(params)
+        if Helper.xcode_at_least?("9")
+          UI.user_error!("resetting simulators currently doesn't work with Xcode 9, stay tuned as we are working to add support for all new tools.")
```

Comment:
But if they are building the app with Xcode 9 and Swift 4 or using any of the new XCTest stuff, it wont work for them if they use `xcode-select` to change to version 8 right?